core/src/main/kotlin/org/evomaster/core/problem/external/service/param/ResponseParam.kt

```diff
@@ -0,0 +1,24 @@
+package org.evomaster.core.problem.external.service.param
+
+import org.evomaster.core.problem.api.service.param.Param
+import org.evomaster.core.search.gene.EnumGene
+import org.evomaster.core.search.gene.ObjectGene
+import org.evomaster.core.search.gene.OptionalGene
+
+
+class ResponseParam (
+    /**
+     * Contains the values for HTTP status codes
+     */
+    val status: EnumGene<Int> = EnumGene("status", listOf(200, 400, 401, 403, 404, 500)),
```

Comment:
avoid code duplication. what is the difference of this list and the one in `ExternalServiceGeneBuilder`? is this latter class even in use?

Reply:
ExternalServiceGeneBuilder serves no purpose now. Will be removed on the next commit.